indexer/feature.cpp

```diff
@@ -345,6 +347,7 @@ void FeatureType::ParseHeader2()
   {
     ptsCount = bitSource.Read(4);
     if (ptsCount == 0)
+      // A mask of outer geometry present.
```

Comment:
Please, put this comments above. A block without {} should have only one line expression.
```
// comment
if (xxx)
  yyy;
```